### ISSUE TITLE
color-contrast accessibility issue

### ISSUE THREAD

**Issue by tballmsft** [microsoft/devicescript#359]
The search bar (upper right) of https://microsoft.github.io/devicescript/ doesn't have sufficient contrast.

![image](https://user-images.githubusercontent.com/10767146/234114002-88b69d8b-62d4-4248-aa6b-9121ecbf384c.png)

**Comment by tballmsft** [comment]
![image](https://user-images.githubusercontent.com/10767146/234114735-20b454c7-224c-45db-9b00-f5b98a0b3e55.png)

**Comment by tballmsft** [comment]
Also the link in copyright.


![image](https://user-images.githubusercontent.com/10767146/234117065-f6391596-9dfe-44c2-b654-0617ed8e9d53.png)

**Comment by tballmsft** [comment]
![image](https://user-images.githubusercontent.com/10767146/234117232-7664a2ba-e3cf-4fa6-8098-601b78ad842b.png)
